code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -49,6 +49,7 @@
 	canSmoothWith = list(/obj/effect/decal/cleanable/dirt, /turf/closed/wall, /obj/structure/falsewall)
 	smooth = SMOOTH_FALSE
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	var/setdirtiness = TRUE

```

Comment:
Seems useless.

Reply:
This is used so that the natural dirt subtype doesn't max out the turf's dirtiness var on init. I don't see any way to remove this var without making natural dirt build-up instantly cause dirtiness to max out once it reaches the threshold to appear.